csharp/msbuild/zeroc.ice.net.props

```diff
@@ -8,33 +8,6 @@
         <IceSoVersion>38</IceSoVersion>
         <IceHome>$(MSBuildThisFileDirectory)..</IceHome>
         <IceAssembliesDir>$(IceHome)\lib</IceAssembliesDir>
+        <IceToolsPath>$(IceHome)\tools</IceToolsPath>
     </PropertyGroup>
-
-    <Choose>
```

Comment:
The properties below were used to find slice2cs in different platforms. With the new package we will always use the Slice compiler included in the NuGet package.

Reply:
In the future when we publish to NuGet Gallery, will be include binaries for Linux, macOS, and Windows?